### DISCUSSION TITLE
can libvips support USM sharpen feature in thumbnail or resize function?

### DISCUSSION THREAD

**Discussion by stonebottle** [libvips/libvips#3926] [General]
USM  algorithm (unsharpen mask) use lanczos2 / lanczos3 for filter。
In some big pictures reduce case, I need this USM for sharpening image details.

**Comment by jcupitt** [comment]
Hi @stonebottle,

The libvips USM is `sharpen`:

https://www.libvips.org/API/current/libvips-convolution.html#vips-sharpen

You'd do something like:

```python
import pyvips

image = pyvips.Image.thumbnail(filename, 256)
image = image.sharpen()
image.write_to_file(output_filename)
```
